packages/core/src/dispatcher.ts

```diff
@@ -157,6 +167,9 @@ export class Dispatcher {
   }
 
   async init() {
+    if (process.env.CERAMIC_ENABLE_V4_MODE) {
```

Comment:
is it necessary to have `CERAMIC_ENABLE_V4_MODE` as a separate variable? could we just rely on `IPFS_FLAVOR='rust'` as we don't really support rust with v3 (it's possible if you have a specific version of ceramic-one, but not planned going forward)

Reply:
I think the `IPFS_FLAVOR` env var only controls behavior of the unit tests currently.  I don't think it has any affect on the runtime behavior of the node, so doing that would be a change to the semantics of that env var.

That said changing the meaning of that env var may be totally fine.  I don't feel super strongly.  This env var is temporary anyway and something that will mostly be for us internally and maybe a handful of partners testing the alpha/beta releases of V4.

I slightly lean towards keeping them separate, but like I said I don't feel strongly, happy to combine them as well if you think that's better.